src/main/java/de/uni_tuebingen/ub/nppm/model/Einzelbeleg.java

```diff
@@ -1,11 +1,16 @@
 package de.uni_tuebingen.ub.nppm.model;
 
+import de.uni_tuebingen.ub.nppm.db.AbstractBase;
```

Comment:
AbstractBase should never be imported (i know sometimes netbeans does add this automatically but it's wrong)